_search-plugins/star-tree-index.md

```diff
@@ -185,6 +189,48 @@ POST /logs/_search
 
 Using a star-tree index, the result will be retrieved from a single aggregated document as it traverses the `status=500` node, as opposed to scanning through all of the matching documents. This results in lower query latency.
 
+#### Date Histogram Aggregation with Metric Aggregations
+
+[Date Histogram]({{site.url}}{{site.baseurl}}/aggregations/bucket/date-histogram/) on calendar intervals with above metric sub-aggregations is supported as of OpenSearch 2.19. 
+
+To use date histogram aggregations searchable via star-tree:
+
+- The calendar intervals in star-tree mapping configuration should have either the request calendar field or a lower granularity calendar field. For example, `month` can be resolved by star-tree from `day` field as well if present in star-tree mapping.
+- Metric sub-aggregation must be part of the aggregation request.
+
+##### Example
+
+The following example gets the sum of all the values in the `size` field aggregated for each calendar month, for all error logs with `status=500`.
+
+```json
+POST /logs/_search
+{
+  "query": {
+    "term": {
+      "status": "500"
```

Comment:
Can we use range query or keyword term query since term is already used in the above example ? Maybe keyword term query will be a good example.

Reply:
@bharath-techie: I'll update the example to use a range query.